recipes/sentencepiece/build.sh

```diff
@@ -0,0 +1,9 @@
+mkdir build
+cd build
+cmake ..
```

Comment:
This should fix things:
```suggestion
cmake 
    -DCMAKE_INSTALL_PREFIX=$PREFIX \
    -DCMAKE_INSTALL_LIBDIR=$PREFIX/lib \
    ..
```